dapps/assets/src/i18n/de.json

```
@@ -30,6 +30,8 @@
       "message": "Nachricht",
       "name": "Name",
       "note": "Notiz",
+      "note-placeholder": "bspw. ein Alias",
```

```suggestion
      "note-placeholder": "Bspw. ein Alias",
```
Lieber konsistent groß schreiben?